src/components/TextForm.tsx

```diff
@@ -20,13 +23,16 @@ export default function ({
     <>
       <Input
         type="text"
+        isError={!!error}
```

Comment:
You're using `!!error` twice, let's extract it into the `const hasError`

Reply:
Done